### ISSUE TITLE
Veture 0.19.0 throwing multiple error messages.

### ISSUE THREAD

**Issue by colinbes** [vuejs/vetur#1237]
- [x] I have searched through existing issues
- [x] I have read through [docs](https://vuejs.github.io/vetur)
- [x] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)

## Info

- Platform: macOs
- Vetur version: 0.19.0
- VS Code version: 1.33.1

## Problem

In code edit, error (red squiggle) on word 'template'

Seeing multiple error messages stating can't find xxx, as example:

```
Cannot find name 'drag'.
Cannot find global type 'BigInt'.
Expected 3 arguments, but got 7.
The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.
```
Have also seen message that my .vue file is not a module.

Changing 
```
<template>
  <form v-on:submit.prevent class="toolbar grid5 drag-disabled">
    <slot></slot>
  </form>
</template>
```
to 
```
<template lang="html">
  <form v-on:submit.prevent class="toolbar grid5 drag-disabled">
    <slot></slot>
  </form>
</template>
```
seems to solve the problem, at least not reporting any errors.

Here's my .vue file

```
<template lang="html">
  <form v-on:submit.prevent class="toolbar grid5 drag-disabled">
    <slot></slot>
  </form>
</template>

<script>
export default {

}
</script>

<style lang="less" scoped>
form.toolbar.grid5 {
  display: grid;
  grid-template-columns: repeat(5, 1fr);
  grid-gap: 0.3em 0.6em;
  grid-auto-flow: dense;
  align-items: center;
}

form.toolbar {
  padding: 5px;
  background-color: @soft-black;
  border-radius: 4px;
  position: absolute;
  width: calc(100% - 30px);
  bottom: 10px;
}
</style>
```

**Comment by kinesias** [comment]
Can confirm! 0.19 breaks EVERYTHING in VS code! this is a main issue! 
I have VS Code Version 1.33.1. 

Some of the error messages I get are: 
- Cannot find global type 'BigInt'.
- The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.

The parser seems to be kaput...

**Comment by octref** [comment]
@kinesias You can either rollback, stop using Vetur, or create a repro case instead of ranting.

**Comment by kinesias** [comment]
Sorry, I didn’t mean to rant. Ventur is awesome, so awesome indeed I’m using it on a daily basis. I just wanted to point out it has nothing to do with OP‘s variables or code. Thanks for your great work! :)

**Comment by octref** [comment]
Seems fixed in 0.19.1. Can you confirm?

![image](https://user-images.githubusercontent.com/4033249/56700516-919ff500-66af-11e9-8e14-3ae51fe2eef5.png)

**Comment by kinesias** [comment]
I'm sorry, I still get the same errors with 0.19.1. here's a screenshot.
the error messages eg "cannot find name 'emplate' seems weird... 
<img width="984" alt="Bildschirmfoto 2019-04-25 um 01 50 22" src="https://user-images.githubusercontent.com/19649063/56700862-a9519a80-66fc-11e9-956f-ff4b64ee828b.png">

**Comment by skydiver** [comment]
![image](https://user-images.githubusercontent.com/578784/56701390-48fb3280-66d5-11e9-9971-4a38120959eb.png)

```
Argument of type '{}' is not assignable to parameter of type 'new (...args: any[]) => any'.
  Type '{}' provides no match for the signature 'new (...args: any[]): any'.
```

**Comment by octref** [comment]
@kinesias https://stackoverflow.com/help/mcve. https://github.com/octref/veturpack

Also once you found a minimal repro case, use the command "Vetur: Show corresponding virtual file and sourcemap" to get the info, and put them here too, thanks.

@skydiver You don't have Vue typing installed probably. `yarn add vue`.

**Comment by joshtaylor0404** [comment]
I am having this issue as well with 0.19.1.

I am getting all kinds of errors in the problems tab of VS code, including the `Argument of type '{}' is not assignable to parameter of type` message mentioned above.

**Comment by cyrrill** [comment]
A new Vue file with only the bare minimal tag:

![image](https://user-images.githubusercontent.com/2376084/56702196-77bedc00-66c8-11e9-883e-de5011ff1078.png)

```
Argument of type '{}' is not assignable to parameter of type 'new (...args: any[]) => any'.
  Type '{}' provides no match for the signature 'new (...args: any[]): any'.Vetur(2345)
```

![image](https://user-images.githubusercontent.com/2376084/56702206-82797100-66c8-11e9-9354-b2b9f93b28ef.png)

Also getting these errors in older existing Vue files:

```
An object literal cannot have multiple properties with the same name in strict mode.Vetur(1117)
An object literal cannot have multiple properties with the same name in strict mode.Vetur(1117)
An object literal cannot have multiple properties with the same name in strict mode.Vetur(1117)
Expected 2 arguments, but got 5.Vetur(2554)
Expected 3 arguments, but got 2.Vetur(2554)
The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.Vetur(2362)
```

I understand you need more info to determine a fix, however, it seems that quite a few people are having issues so I would consider this a confirmed bug.

Thanks for you work on Vetur, it's an amazing extension! :)

**Comment by kenberkeley** [comment]
Same issue here
![image](https://user-images.githubusercontent.com/13287948/56702327-b72c0d80-6746-11e9-8037-746c5f41a1aa.png)

```
{
	"resource": "/c:/Users/Ken/Desktop/vue-demo/src/pages/stocks/index.vue",
	"owner": "_generated_diagnostic_collection_name_#1",
	"code": "2345",
	"severity": 8,
	"message": "Argument of type '{}' is not assignable to parameter of type 'new (...args: any[]) => any'.\n  Type '{}' provides no match for the signature 'new (...args: any[]): any'.",
	"source": "Vetur",
	"startLineNumber": 1,
	"startColumn": 1,
	"endLineNumber": 1,
	"endColumn": 1
}
```

**Comment by skydiver** [comment]
my workaround was to disable this option:
`"vetur.experimental.templateInterpolationService": false`

so, it must be related to that feature

**Comment by cyrrill** [comment]
Hi @octref , here is the virtual file you asked for:

(*only edit: hidden local path names for project privacy)

```
====================
Virtual content of /home/username/***/components/input/AddressInput.vue.template
Hover, semantic diagnostics, jump to definition and find references are run on this file.
====================

import __Component from "./AddressInput.vue";
import { __vlsRenderHelper, __vlsComponentHelper, __vlsIterationHelper, __vlsListenerHelper } from "vue-editor-bridge";
__vlsRenderHelper(__Component, function () {
    "\n    ";
    __vlsComponentHelper("div", { props: {}, on: {}, directives: [] }, ["\n\n    "]);
    "\n           \n\n        \n                \n\n \n         \n\n              \n\n        ";
});


====================
SourceMap
from: /home/username/***/components/input/AddressInput.vue
to  : /home/username/***/components/input/AddressInput.vue.template
[VueFileStart, VueFileEnd, VueFileText] => [TSVirtualFileStart, TSVirtualFileEnd, TSVirtualFileText]
====================
```

**Comment by cyrrill** [comment]
Interesting note:

If I start with an empty file, and add <script> & <style> tags, all good, no errors.

As soon as I add a <template> tag to the file, it throws the error.

Interestingly, a template tag such as this:

```
<template lang="html">
</template>
```

or say with lang="pug", throws no exception. It seems to be only a bare `<template>` tag which is problematic.

**Comment by dan2600** [comment]
having the same issue in vc code here are my errors

```
Expected 2 arguments, but got 3. Vetur2554
Expected 3 arguments, but got 6 Vetur2554
Cannot find name 'emplate'.  Vetur2304
Cannot find name 'on'. Vetur2304
Cannot find name 'directives'. Vetur2304
```

**Comment by colinbes** [comment]
Adding lang=“html” definitely clears errors as reported. Gotta be a hint there.

**Comment by ntnyq** [comment]
Got same issue this morning with Vetur v0.19.1 and skydiver's solution works for me.
Thanks.

**Comment by dan2600** [comment]
my main app.vue doesn't need it. but adding lang="html" to my components fixes the issue for me as well. Thanks!

**Comment by MarkSky** [comment]
After updating from 0.19.0 to 0.19.1, I got the error:
![VETUR](https://user-images.githubusercontent.com/4032734/56704755-f5bec980-6740-11e9-808d-4720454db0fd.jpg)
error message:

> Argument of type '{}' is not assignable to parameter of type 'new (...args: any[]) => any'.
  Type '{}' provides no match for the signature 'new (...args: any[]): any'.Vetur(2345)

They  happened in every .vue files

**Comment by lincenying** [comment]
v0.19.1

![image](https://user-images.githubusercontent.com/10840991/56705070-0facdc00-6742-11e9-84ee-ebe351f01e45.png)

![image](https://user-images.githubusercontent.com/10840991/56705108-2e12d780-6742-11e9-8523-62181fdc7979.png)

**Comment by wangdahoo** [comment]
> Interesting note:
> 
> If I start with an empty file, and add <script> & <style> tags, all good, no errors.
> 
> As soon as I add a `<template>` tag to the file, it throws the error.
> 
> Interestingly, a template tag such as this:
> 
> ```
> <template lang="html">
> </template>
> ```
> or say with lang="pug", throws no exception. It seems to be only a bare `<template>` tag which is problematic.

work for me, thanks

**Comment by peps** [comment]
I just rolled back to 0.18.1 and everything works perfectly again for me. 0.19.1 doesn't seem to be stable at the moment...

**Comment by zhangenming** [comment]
19.1
`[Error - 上午10:33:02] Request textDocument/definition failed.
  Message: Request textDocument/definition failed with message: Cannot read property 'Symbol(Symbol.iterator)' of undefined
  Code: -32603 `

**Comment by yinxulai** [comment]
![image](https://user-images.githubusercontent.com/22045569/56705881-62d45e00-6745-11e9-859e-219999a0c44a.png)
I also met

**Comment by bbbford** [comment]
Same issues here guys. Had to roll back to 0.18.1 and everything is back to working.

**Comment by Gyufei** [comment]
i hava same problem in vetur `0.19.0`, and vscode `1.33.1` 。
when i set `"vetur.validation.template": false` , all good。
![image](https://user-images.githubusercontent.com/25930828/56707397-3a4f6280-674b-11e9-8300-19a9389bce53.png)

**Comment by chanlito** [comment]
The PR in #1222 doesn't fix @yinxulai https://github.com/vuejs/vetur/issues/1237#issuecomment-486497996 case

**Comment by larsmars** [comment]
I have the same:  on <templates> : An object literal cannot have multiple properties with the same name in strict mode.Vetur(1117)

**Comment by lincenying** [comment]
rolled back to 0.18.1

1. remove 0.19.1

2. download 0.18.1
https://marketplace.visualstudio.com/_apis/public/gallery/publishers/octref/vsextensions/vetur/0.18.1/vspackage

3. install
![image](https://user-images.githubusercontent.com/10840991/56711319-d3d34000-675c-11e9-8a6e-3e290d1bdc4b.png)

**Comment by vadimjoy** [comment]
**Vetur v0.19.1** VSCode v1.33.1
![image](https://user-images.githubusercontent.com/14856940/56711420-28b68c80-6744-11e9-808d-5bdd9270ba72.png)

![image](https://user-images.githubusercontent.com/14856940/56711401-13d9f900-6744-11e9-9353-a70c10423e35.png)

![image](https://user-images.githubusercontent.com/14856940/56711447-4c79d280-6744-11e9-9ece-315b7ca781f4.png)

**Comment by ktsn** [comment]
@vadimjoy That's not related to this issue. -> #1203

**Comment by diegootayza** [comment]
in the documentation is the solution:
[https://vuejs.github.io/vetur/interpolation.html#generic-language-features](url)

**Comment by ktsn** [comment]
I found the following cases generate number of weird errors in template (at Vetur 0.19.1):

1. Using template literal (```:value="`foo: ${foo}`"```) -> #1230, #1244 
2. Including hyphen in some object literal property name (`:class="{ 'hyphen-class': true }"`)

I'm fixing these cases now.
If there are other case which causes this kind of issue, please comment it with reproducible `.vue` file. Thanks for your cooperation!

**Comment by codesboy** [comment]
The same problem in vetur 0.19.1

![E7417C4F-0B3B-4A94-9B86-34BE5A68EC1D](https://user-images.githubusercontent.com/9047357/56715483-64fce380-676a-11e9-8e3b-0d1353a131b7.png)

**Comment by dodanex** [comment]
Got the same issue with v0.19.1. File ...vue. is not a module. If I add `<template lanh="html">` then all is ok, but this is not a good fix to add it to al my files and in all my projects. Hope will be fixed soon.

**Comment by karl93rus** [comment]
Same issue with 0.19.1. :(

**Comment by basilky** [comment]
Same problem

**Comment by timo-jj** [comment]
Hey Peeps,
I had the same issue as well.
I think for the moment we can switch to the version 0.18.1 of Vetur.
Do not forget to disable the plugins automatic update on VSCode.
It could be a temporary solution until version 0.19.1 is fixed.

**Comment by godarren** [comment]
Same on Windows with vscode and Vetur. I downgraded but had to disable auto update for extensions because it just updated again the next restart... Hopefully it's fixed soon and can return to standard settings. I can confirm 0.18.1 works fine but update breaks.

**Comment by cyrrill** [comment]
Hi all,

Earlier I mentioned that `<template lang="html">` would sort the issue, that is so. However, I only mentioned that as a debugging clue for the maintainers. It was not meant to be a solution, and I personally have not done so in my code.

It is not a good idea to go and "pollute" all your code with unnecessary markup just to palliate this bug.

Additionally, it is not necessary to downgrade, or much less to disable auto-update in Code, that can cause other issues.

The best way to deal with this while the maintainers of the extension fix this, as noted by @skydiver, is to simply disable the experimental feature which got added in the latest release, by going into:

File > Preferences > (search for "vetur"), and then disable this checkbox:

![image](https://user-images.githubusercontent.com/2376084/56741861-46322900-6739-11e9-911e-40f1274de6d6.png)

@octref @ktsn in the future please do not enable "experimental" features by default. It's ok to ship it so we can test it, but ship it in an optional and disabled manner so that untested code doesn't wreak havoc as seen here.

Thanks!

**Comment by dernest92** [comment]
> I'm getting these errors on every .vue file
> 
> ![image](https://user-images.githubusercontent.com/578784/56701390-48fb3280-66d5-11e9-9971-4a38120959eb.png)
> 
> ```
> Argument of type '{}' is not assignable to parameter of type 'new (...args: any[]) => any'.
>   Type '{}' provides no match for the signature 'new (...args: any[]): any'.
> ```

I was having the same issue

Adding the language attribute fixed it for me.
`<template lang="html">`

**Comment by octref** [comment]
Please stop misleading people by saying adding `lang="html"` fixes the problem.

If you have read the docs you would find there's a setting to turn this off:

![image](https://user-images.githubusercontent.com/4033249/56749036-cc4c7080-6735-11e9-9208-dc740d9059b7.png)

You can also use `vetur.validation.template` false to disable all template validation.

**Comment by cyrrill** [comment]
@octref  No one is purposely misleading anyone. However, you do seem to have an accusative tone which is unproductive.

Instead of releasing broken software and expecting people to dig through the docs of an obscure extension in their IDE (who does reads that?!), better test what you code, and if you decide to release "experimental" features, don't turn them on by default and expect everyone to figure out why it's broken.

**Comment by outofmemoryagain** [comment]
@cyrrill, I understand your frustration.  I'm not a Vetur contributor, BUT keep in mind this is open source software.  And the contributors have provided superb work that makes all of our lives easier, but bugs do happen especially when making significant changes to the foundation of an extension such as this one.  Open source projects do not have infinite resources to test every permeation of every project and development environment.  This is the price will all pay for fast release cycles and responsive releases and free software.  In the end these features will eventually make our lives easier and more productive.

**Comment by alexcroox** [comment]
Note to others, unchecking `vetur.experimental.templateInterpolationService` did not have an immediate effect, restarting vscode after turning it off fixes all issues described above.

**Comment by cyrrill** [comment]
@outofmemoryagain , I totally get that, my response comes to the developer's attitude, not the product. Besides, octref is not a random starving opensource developer, he's at Microsoft, where professionalism would be expected.

If you read: 

https://github.com/vuejs/vetur/issues/1237#issuecomment-486460904 
https://github.com/vuejs/vetur/issues/1237#issuecomment-486726769

You'll notice a dismissive and harsh tone, which is not productive at all in collaborative projects.

Anyways, this is veering way off-topic. It's clear what the bug is, and a solution is probably soon to come.

**Comment by tiagodurante** [comment]
![image](https://user-images.githubusercontent.com/5533291/56776616-c68d7400-67a3-11e9-87bd-c58525275b5a.png)
![image](https://user-images.githubusercontent.com/5533291/56776632-d907ad80-67a3-11e9-8697-82146a3c81f3.png)

rollback version to 0.18.1 but, when i reload my vscode, this package is updated. 😰

**Comment by zhangkun95** [comment]
Change vetur settings
Uncheck the following
Validation: Template
Validate vue-html in <template> using eslint-plugin-vue

**Comment by chanlito** [comment]
@zhangkun95 you will lose all of eslint-plugin-vue goodies, the solution is already mentioned above.

**Comment by zhangkun95** [comment]
> @ zhangkun95你将失去所有的eslint-plugin-vue好东西，上面已经提到了解决方案。

thank you for your reply

**Comment by octref** [comment]
> No one is purposely misleading anyone. However, you do seem to have an accusative tone which is unproductive.

I'm accusative because adding `lang="html"`:

- Requires users to change their code (which they shouldn't, it's a Vetur bug)
- Disables all other Vetur feature for template region

I don't want to solve new issues that says "hey why emmet doesn't work in my <template lang="html"> region".

> Instead of releasing broken software and expecting people to dig through the docs of an obscure extension in their IDE (who reads that?!)

Opening issue asks you to tick this box.

![image](https://user-images.githubusercontent.com/4033249/56778835-37756180-678d-11e9-9001-2bd3ccaca132.png)

>  if you decide to release "experimental" features, don't turn them on by default and expect everyone to figure out why it's broken.

Then no one will find it and use it. You just said "no one would dig through the docs of an obscure extension in their IDE".

Once this feature matures it'll be a major differentiator for Vue over React (which relies on TypeScript for language features) or Angular (whose AOT is what many people want for their TS projects, but its language features aren't working well today). So I'd like to turn it on, iterate and improve. If you don't like it, there are multiple ways to opt-out.

> Besides, octref is not a random starving opensource developer, he's at Microsoft, where professionalism would be expected.

![image](https://user-images.githubusercontent.com/4033249/56779279-16ae0b80-678f-11e9-92db-6e065a57ab3d.png)

If you aren't giving a repro and yelling at me "EVERYTHING IS BROKEN", there's nothing I can help so I'll be curt. I got limited time and hundreds of issues in my inbox. You should read https://nolanlawson.com/2017/03/05/what-it-feels-like-to-be-an-open-source-maintainer/ because that's how I feel.

In short:

- Yelling helps no one. Open non-duplicate issues with minimal reproducible case and I'll be able to help.
- I had to stop people from reading "oh adding `lang="html` is the solution!" because clueless people might follow that and open even more invalid issues.

**Comment by GzhiYi** [comment]
Open VsCode settings, search the key 'Auto Update'  and uncheck it.
Search the extension Vetur, pick install another version[0.18.1].

**Comment by octref** [comment]
The original issue is when attributes have `-` in it, Vetur compiles it to invalid TypeScript.

This should be fixed in 0.19.1. If you have new issues, open new issues.

**Comment by fernandoagarcia** [comment]
> > No one is purposely misleading anyone. However, you do seem to have an accusative tone which is unproductive.
> 
> I'm accusative because adding `lang="html"`:
> 
> * Requires users to change their code (which they shouldn't, it's a Vetur bug)
> * Disables all other Vetur feature for template region
> 
> I don't want to solve new issues that says "hey why emmet doesn't work in my region".
> 
> > Instead of releasing broken software and expecting people to dig through the docs of an obscure extension in their IDE (who reads that?!)
> 
> Opening issue asks you to tick this box.
> 
> ![image](https://user-images.githubusercontent.com/4033249/56778835-37756180-678d-11e9-9001-2bd3ccaca132.png)
> 
> > if you decide to release "experimental" features, don't turn them on by default and expect everyone to figure out why it's broken.
> 
> Then no one will find it and use it. You just said "no one would dig through the docs of an obscure extension in their IDE".
> 
> Once this feature matures it'll be a major differentiator for Vue over React (which relies on TypeScript for language features) or Angular (whose AOT is what many people want for their TS projects, but its language features aren't working well today). So I'd like to turn it on, iterate and improve. If you don't like it, there are multiple ways to opt-out.
> 
> > Besides, octref is not a random starving opensource developer, he's at Microsoft, where professionalism would be expected.
> 
> ![image](https://user-images.githubusercontent.com/4033249/56779279-16ae0b80-678f-11e9-92db-6e065a57ab3d.png)
> 
> If you aren't giving a repro and yelling at me "EVERYTHING IS BROKEN", there's nothing I can help so I'll be curt. I got limited time and hundreds of issues in my inbox. You should read https://nolanlawson.com/2017/03/05/what-it-feels-like-to-be-an-open-source-maintainer/ because that's how I feel.
> 
> In short:
> 
> * Yelling helps no one. Open non-duplicate issues with minimal reproducible case and I'll be able to help.
> * I had to stop people from reading "oh adding `lang="html` is the solution!" because clueless people might follow that and open even more invalid issues.

@octref The Vue community appreciates all the hard work you have done maintaining this plugin. It's the one we all use for Vue. No one is impervious to bugs or stress. The plugin speaks for itself of the level of professionalism you have and hope no one else doubts your integrity.

**Comment by cyrrill** [comment]
> Yelling helps no one. 

Says the super polite guy who responds to a bug report:

![image](https://user-images.githubusercontent.com/2376084/56783446-68678d80-67b1-11e9-8382-9bb2ea7ee1c0.png)


> I'm accusative because adding lang="html":

No has ever suggested to use `lang="html"`, it was only mentioned as a debugging clue, the post immediately preceding your mentions this explicitly:

![image](https://user-images.githubusercontent.com/2376084/56783545-e0ce4e80-67b1-11e9-889a-f9d1f578b447.png)


I don't see the point in saying people are "misleading", that's a strong term which implies dishonesty.

> mislead: verb ˌmis-ˈlēd: to lead in a wrong direction or into a mistaken action or belief often by deliberate deceit


You can be busy, intelligent, and accomplished;  it doesn't entitle you to be rude or accusative. You call it curt, I call that being a dick.

**Comment by octref** [comment]
@cyrrill Stop taking conversations out of their context, please? That was in response to @kinesias's comment, not to the OP.

![image](https://user-images.githubusercontent.com/4033249/56783858-9d201880-67a2-11e9-8f4b-3a1366e14b00.png)

> I don't see the point in saying people are "misleading", that's a strong term which implies dishonesty.

I'm not aware of that implication. Maybe it's because I'm not a native speaker. I only meant "please don't lead people to wrong solution", in the spirit of not increasing invalid bugs reports. I was already getting 20+ issues and I didn't want more invalid ones.

I didn't imply dishonesty from you. If that's how you interpreted it, sorry. I could have used some words like "That's not the right solution", but I had many people coming into these issues and I had to stop them from following the wrong advice.

**Comment by cyrrill** [comment]
@octref, the context is quite clear for anyone reading the thread. 

No one gets 40 :-1:  on their comment without merit, so please don't pretend like you're so innocent and oblivious to your crassness. 

You can write the most amazing software, but if you treat people dismissively and rudely, nothing has been gained.

The basic point here is: treat people with respect, and your life as a maintainer (and in general) will be much easier.

.

**Comment by octref** [comment]
I treat @colinbes's bug report with respect.

@kinesias's first response is:
- Not related to original issue
- Doesn't contain repro case
- Has all caps yelling
- Adds nothing to OP's bug report and dilutes content

So I was being curt. Imagine you are a doctor in the middle of diagnosing a patient and another patient blazing in yelling to you "I have problems too!", what would you do? I'd get him out of my office and send him to front desk to make an proper appointment for diagnosis. **I don't think this is disrespect.**

I did follow up with this, when he tries to create a repro case.

![image](https://user-images.githubusercontent.com/4033249/56787276-dcedfc80-67b0-11e9-815e-11fec2c761c7.png)

> The basic point here is: treat people with respect, and your life as a maintainer (and in general) will be much easier.

I treat all issues this way:

- Have repro? Fix
- No repro? Ask for repro
- Still no repro? Close or ignore

You can read https://nolanlawson.com/2017/03/05/what-it-feels-like-to-be-an-open-source-maintainer/ to learn more about what the real life of a maintainer is like. Before you start to have 200 notifications in your GitHub inbox, please don't try to lecture to others how to be a maintainer.

**Comment by octref** [comment]
Locking since neither me nor any more commenters add anything meaningful to the original issue, which has been fixed and released in 0.19.1.

If you have issues open new issues **with valid reproducible case**.
